include/flatbuffers/flatbuffers.h

```diff
@@ -69,7 +69,7 @@ template<typename T> T EndianSwap(T t) {
   }
 }
 
-template<typename T> size_t AlignOf() {
+template<typename T> constexpr size_t AlignOf() {
```

Comment:
This is used in the reallocate func, where the alignment is marked as a constexpr. I saw that there were already preprocessor checks for a constexpr fallback to const, so I thought we'd already handled portability for older compilers?